test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -973,7 +973,7 @@ TEST(nntrainerModels, read_save_01_n) {
 
 TEST(nntrainerModels, loadFromLayersBackbone_p) {
   std::vector<std::shared_ptr<ml::train::Layer>> reference;
-  reference.emplace_back(ml::train::layer::FullyConnected({"name=fc1"}));
+  reference.emplace_back(ml::train::layer::FullyConnected({"name=fc1", "input_shape=3:1:2"}));
```

Comment:
this unittest should work without the input_shape as well, as fc1 is listed as start layer?

Reply:
This was initially wrong.

This layer becomes the input of the model, so either this has to be stated to be input of the model (even in this case we need input shape defined).

Reply:
cool.